src/opsgenie.ts

```diff
@@ -269,16 +277,19 @@ export const closeOpsGenieAlertWithAlias = async (alias: string, opsGenieConfig:
     .map((rejection) => log('Alert close promise rejected', 'ERROR', rejection));
 };
 
-export const sendToOpsGenieLowLevel = async (message: OpsGenieMessage, opsGenieConfig: OpsGenieConfig) => {
+export const sendToOpsGenieLowLevel = async (message: OpsGenieMessage, opsGenieConfig = DEFAULT_OPSGENIE_CONFIG) => {
   log(message.message, 'INFO', message);
   if (checkForOpsGenieApiKey()) {
     return;
   }
   const url = 'https://api.opsgenie.com/v2/alerts';
   const apiKey = process.env.OPSGENIE_API_KEY ?? opsGenieConfig.apiKey;
 
+  const hashedAlias = generateHash(message.alias);
```

Comment:
If we generate the hash from the `message.alias` this still requires that the `alias` we set from downstream apps is unique, right? For example, the `alias` would need to include the `chainId` or `chainName` for a chain-related alert or a `txHash` for an event related alert.

Reply:
Yes, definitely, the aliases still need to be unique.
I don't think there's any way of generating predictable aliases programmatically; like you could maybe use the file+line from the source map to generate something unique, but it wouldn't help for something like a loop that collects events from different chains :/

Reply:
Just FYI I've reverted the hashed alias change as it appears Opsgenie recently relaxed that limit.
The package is published and it's working well in another PR https://github.com/api3dao/operations-database/pull/347